src/main/java/com/englishtown/bitbucket/hook/MirrorRequest.java

```diff
@@ -0,0 +1,33 @@
+package com.englishtown.bitbucket.hook;
+
+import com.atlassian.bitbucket.repository.Repository;
+
+import java.io.Serializable;
+
+class MirrorRequest implements Serializable {
```

Comment:
`MirrorRequest` and `MirrorSettings` must be `Serializable` to work with the `BucketedExecutor`--they may be serialized and moved between nodes in a cluster.